src/app/page.tsx

```diff
@@ -28,8 +30,44 @@ const cardTwoContent = (
 	</ul>
 	// {/* </div> */}
 );
+interface Testimonial {
+	id: number;
+	created_at: string;
+	review_content: string;
+	user_id: number;
+	userName: string;
+	user_profile_link: string;
+  }
+  
+function returnNewCard(testimonial : Testimonial ){
```

Comment:
Move component to a component file under the @/components directory.

Reply:
Done